.github/workflows/integration-tests.yml

```
@@ -1,240 +1,26 @@
-          
-          # Webhook secrets are expected in test files, so only check for production patterns
-          if grep -r "whsec_[a-zA-Z0-9]{32,}" . --exclude-dir=node_modules --exclude-dir=tests --exclude="*.test.js" --exclude="*.md" 2>/dev/null | grep -v "test" | grep -v "mock"; then
-            echo "⚠️ Found potential webhook secret (may be test fixture)"
-          fi
-          
-          echo "✅ No production secrets found in code"
\ No newline at end of file
+          echo "❌ This workflow is deprecated!"
+          echo "Please use 'Post-Merge Validation' workflow instead."
+          echo "Location: .github/workflows/post-merge-validation.yml"
+          exit 1
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)